rust/cubesql/cubesql/src/transport/service.rs

```diff
@@ -113,6 +113,12 @@ impl SpanId {
     }
 }
 
+#[derive(Clone)]
+pub struct TransportServiceLoadResponse {
```

Comment:
At least, it's incorrect name for such structure in term of the usage, because it wraps RecordBatch, but result of the `load` method is `Vec<TransportServiceLoadResponse>`. It just missleading, let's name `TransportServiceLoadRecordBatch`

Annother question, what is the reason to wrap RecordBatch instead of storing `last_refresh_time` at top level?

```
TransportServiceLoadResponse {
     pub last_refresh_time: Option<String>,
     pub batches: Vec<RecordBatch>,
}
```